src/components/FeatureList/FeatureList.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Flex, View, Heading } from '@aws-amplify/ui-react';
+
+interface FeatureListProps {
+  heading: string;
+  level: 1 | 2 | 3 | 4 | 5 | 6;
+  children: React.ReactNode;
+}
+const FeatureList: React.FC<FeatureListProps> = ({
+  children,
+  heading,
+  level
+}) => {
+  console.log('heading', heading);
+  return (
+    <>
+      <Flex
+        direction="column"
+        justifyContent="flex-start"
+        alignItems="stretch"
+        alignContent="flex-start"
+        wrap="nowrap"
+        gap="1rem"
+        paddingRight={'xxl'}
+        as="ul"
```

Comment:
```suggestion
```
If you take a look at your output HTML, you have some improper nesting going on:
```
ul
  h3
  ul
```
I'd use a div for this element instead.